src/testUtils/java/uk/gov/hmcts/reform/wataskmanagementapi/SpringBootFunctionalBaseTest.java

```diff
@@ -183,78 +206,128 @@ public String createTask(CreateTaskMessage createTaskMessage) {
+                                Jurisdiction jurisdiction) {
+        Headers headers = getAuthHeadersForJurisdiction(jurisdiction);
+        initiateTask(testVariables, headers, null, defaultInitiationAssert(testVariables));
+    }
 
-        ZonedDateTime createdDate = ZonedDateTime.now();
-        String formattedCreatedDate = CAMUNDA_DATA_TIME_FORMATTER.format(createdDate);
-        ZonedDateTime dueDate = createdDate.plusDays(1);
-        String formattedDueDate = CAMUNDA_DATA_TIME_FORMATTER.format(dueDate);
+    protected void initiateTask(TestVariables testVariables,
+                                Headers headers) {
+        initiateTask(testVariables, headers, null, defaultInitiationAssert(testVariables));
+    }
 
-        InitiateTaskRequest req = new InitiateTaskRequest(INITIATION, asList(
-            new TaskAttribute(TASK_TYPE, taskType),
-            new TaskAttribute(TASK_NAME, taskName),
-            new TaskAttribute(TASK_TITLE, taskTitle),
-            new TaskAttribute(TASK_CASE_ID, testVariables.getCaseId()),
-            new TaskAttribute(TASK_CREATED, formattedCreatedDate),
-            new TaskAttribute(TASK_DUE_DATE, formattedDueDate)
-        ));
+    protected void initiateTask(TestVariables testVariables,
+                                Jurisdiction jurisdiction,
+                                Consumer<Response> assertConsumer) {
+        Headers headers = getAuthHeadersForJurisdiction(jurisdiction);
+        initiateTask(testVariables, headers, null, assertConsumer);
+    }
 
-        Response result = restApiActions.post(
-            TASK_INITIATION_ENDPOINT,
+    protected void initiateTask(TestVariables testVariables,
+                                Headers headers,
+                                Consumer<Response> assertConsumer) {
+        initiateTask(testVariables, headers, null, assertConsumer);
+    }
+
+    protected void initiateTask(TestVariables testVariables,
+                                Jurisdiction jurisdiction,
+                                Map<String, String> additionalProperties) {
+        Headers headers = getAuthHeadersForJurisdiction(jurisdiction);
+        initiateTask(testVariables, headers, additionalProperties, defaultInitiationAssert(testVariables));
+    }
+
+    protected void initiateTask(TestVariables testVariables,
+                                Headers headers,
+                                Map<String, String> additionalProperties) {
+        initiateTask(testVariables, headers, additionalProperties, defaultInitiationAssert(testVariables));
+    }
+
+    private void initiateTask(TestVariables testVariables,
+                              Headers headers,
+                              Map<String, String> additionalProperties,
+                              Consumer<Response> assertConsumer) {
+        log.info("Task initiate with cron job {}", initiationJobRunning);
+        if (initiationJobRunning) {
```

Comment:
initiation process has to wait for task initiation job in AAT, I guess we need to wait for task initiation and then check it's in CFT task DB. If I'm wrong please let me know. This is my suggestion for a delay
this waits 2 seconds before DB check

with()
                .pollInterval(2, SECONDS)
                .and()
                .with()
                .pollDelay(2, SECONDS).
                await()
                .atMost(60, SECONDS)
                .until(
                    () -> {
                        Response response = restApiActions.get(
                            TASK_GET_ENDPOINT,
                            testVariables.getTaskId(),
                            headers
                        );

                        return HttpStatus.OK.value() == response.getStatusCode();
                    }
                );